etrago/appl.py

```diff
@@ -392,10 +405,10 @@ def etrago(args):
     network.storage_units.cyclic_state_of_charge = True
 
     # set extra_functionality
-    if args['extra_functionality'] is not None:
+    """if args['extra_functionality'] is not None:
```

Comment:
It does not seem very nice to keep these lines as a commentary , does it?

Reply:
Yes, you're right. I'll delete these lines.